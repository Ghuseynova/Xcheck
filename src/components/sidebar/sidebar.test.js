import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { shallow } from 'enzyme';
import Sidebar from './sidebar';

describe('render', () => {    
  test('Sidebar', () => {
    const userInfo = {
      avatarUrl: 'https://avatars0.githubusercontent.com/u/14919341?v=4',
      name: 'Maria',
      roles: ['student'],
    };
    const result = shallow(
      <Router>
        <Sidebar user={userInfo} />
      </Router>
    );
    expect(result).toMatchSnapshot()
  });
});
