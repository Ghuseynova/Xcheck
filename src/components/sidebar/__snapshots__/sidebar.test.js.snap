// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Sidebar 1`] = `ShallowWrapper {}`;
